cinema/serializers.py

```diff
@@ -1 +1,67 @@
-# write serializers here
+from rest_framework import serializers
+from .models import Actor, CinemaHall, Genre, Movie, MovieSession, Order, \
+    Ticket
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["id", "first_name", "last_name", "full_name"]
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["id", "name"]
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Movie
+        fields = ["id", "title", "description", "duration", "genres", "actors"]
+
+
+class MovieListSerializer(MovieSerializer):
+    genres = serializers.SlugRelatedField(
+        many=True, read_only=True, slug_field="name"
+    )
+    actors = serializers.SlugRelatedField(
+        many=True, read_only=True, slug_field="full_name"
+    )
```

Comment:
The `SlugRelatedField` with `slug_field='full_name'` is incorrect because `full_name` is a property, not a model field. Use a `SerializerMethodField` to serialize the actor's full name.